q2_diversity_lib/tests/test_alpha.py

```diff
@@ -0,0 +1,55 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2018-2019, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from unittest import TestCase, main
+from q2_diversity_lib import (faith_pd, )
+
+import io
+import biom
+import skbio
+import numpy as np
+import pandas as pd
+import pandas.util.testing as pdt
+
+
+class AlphaTests(TestCase):
```

Comment:
Can you use the QIIME 2 test runner, instead? `from qiime2.plugin.testing import TestPluginBase`

You can see a usage example here: https://github.com/qiime2/q2-vsearch/blob/master/q2_vsearch/tests/test_join_pairs.py

One advantage is that there are some helpers in here for loading data (see that link above for an example, also the other tests in that plugin are good to look at, too) --- this can reduce some boilerplate around construction of your test tables.

The testing util isn't really _that_ great here (most of its helpers are for testing transformers and formats), but might as well start off using it, anyway.

Reply:
✓